src/esperanza/finalizationstate.cpp

```diff
@@ -129,7 +128,7 @@ Result FinalizationState::InitializeEpoch(blockchain::Height blockHeight) {
     log_msg = " epoch=" + std::to_string(m_current_epoch - 2) + " was not justified.";
   }
 
-  LogPrint(BCLog::FINALIZATION,
+  LogPrint(BCLog::FINALIZATION, /* Continued */
```

Comment:
Is this a new convention or what? What's the reason for it?

Reply:
`lint-logs.sh` checks that all log entries end with a newline: it searches the code for `LogPrint*` lines and seeing if it contains `\n"`. If a source line ends with `/* Continued */`, the script also considers the next one.
I do think it looks ugly, but it did detect a few issues in our code already.